src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -85,10 +85,10 @@
      </ItemGroup>
 
     <ItemGroup Condition="'$(CoreLibRoot)' != ''">
-      <TestCopyLocal Include="$(CoreLibRoot)\mscorlib.dll" />
-      <TestCopyLocal Include="$(CoreLibRoot)\System.Private.CoreLib.dll" />
-      <TestCopyLocal Include="$(CoreLibRoot)\PDB\mscorlib.pdb" />
-      <TestCopyLocal Include="$(CoreLibRoot)\PDB\System.Private.CoreLib.pdb" />
+      <RunTestsForProjectInputs Include="$(CoreLibRoot)\mscorlib.dll" />
+      <RunTestsForProjectInputs Include="$(CoreLibRoot)\System.Private.CoreLib.dll" />
+      <RunTestsForProjectInputs Include="$(CoreLibRoot)\PDB\mscorlib.pdb" />
+      <RunTestsForProjectInputs Include="$(CoreLibRoot)\PDB\System.Private.CoreLib.pdb" />
```

Comment:
If these are indeed needed as incremental build inputs to RunTestsForProject, then I think they should be declared in the DiscoverTestInputs target in test.targets alongside the other `RunTestsForProjectInputs` items.